Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -149,9 +163,10 @@ private class ParameterSetNames
         /// The name of the bucket to upload to. Will also accept a Bucket object.
         /// </para>
         /// </summary>
-        [Parameter(Position = 0, Mandatory = true)]
+        [Parameter(Position = 2, Mandatory = false)]
```

Comment:
Is this cmdlet GA? Do the positional and mandatory changes need to happen in a new version as to not break users?